config/initializers/inflections.rb

```diff
@@ -1,16 +1,16 @@
-# Be sure to restart your server when you modify this file.
```

Comment:
You have some mysterious diffs: files you didn't really change and yet they still show up here. Just ignore them for now (or try to get rid of them in this branch and copy paste them from master branch - good activity to practice some branch switching).